doc/adr/0018-indexers-query-synchronisation-primitive.rst

```diff
@@ -0,0 +1,56 @@
+.. _hearing_hysterical_screams:
```

Comment:
git seems to be confused by filename case?
I renamed `0018-Indexers-query-synchronisation-primitive.rst` -> `0018-indexers-query-synchronisation-primitive.rst` (lowercase `i`)
But both files are showing up...

Reply:
try `git mv` ?